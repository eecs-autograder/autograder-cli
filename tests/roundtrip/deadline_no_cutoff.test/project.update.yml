project:
  name: Test Project
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    deadline:
      cutoff_type: none
      deadline: Dec 2, 2024 11:59pm
