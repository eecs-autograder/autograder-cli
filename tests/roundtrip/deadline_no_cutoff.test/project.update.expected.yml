project:
  name: Test Project
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    deadline:
      cutoff_type: none
      deadline: Dec 02, 2024 11:59PM
